stress/src/throughput.rs

```diff
@@ -155,7 +155,7 @@ impl<'a> UnsafeSlice<'a> {
     #[inline(always)]
     unsafe fn increment(&self, i: usize) {
         let value = self.slice[i].get();
-        (*value).count = (*value).count + 1;
```

Comment:
lets remove all unrelated changes, to make it easier on reviewers.
(Its totally fine to send a short, separate PR for this.)